react-native-async-storage/async-storage
# chore(web): Fix start:web

## Summary

The example app is currently broken due to `@react-native-async-storage/async-storage` not being resolved correctly.

Resolves #472.

## Test Plan

1. Run `yarn`.
2. Run `yarn start:web`.
3. Once the dev server is running, press `w` to open the project in browser.
4. The example should work as expected.
